lib/ruby_lsp_rails/railtie.rb

```diff
@@ -0,0 +1,21 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "rails"
+
+module RubyLsp
+  module Rails
+    class Railtie < ::Rails::Railtie
+      config.ruby_lsp_rails = ActiveSupport::OrderedOptions.new
+      config.ruby_lsp_rails.server = true
+
+      initializer "ruby_lsp_rails.setup" do |_app|
+        config.after_initialize do |app|
+          config.ruby_lsp_rails.server
+            raise "not needed!"
```

Comment:
Let's use a deprecation warning from Active Support instead. Otherwise, this will still break people's servers.

We can mention in the warning that the configuration can be removed.